### ISSUE TITLE
Add ACI termination error to error handling

### ISSUE THREAD

**Issue by cmgtam** [uw-cmg/MAST#432]
Zhewen reports, and I have seen:

1) I think this error is not from vasp, but from the job submission to the aci cluster. The error message in the vasp.out is something like
===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   EXIT CODE: 30
=   CLEANING UP REMAINING PROCESSES
=   YOU CAN IGNORE THE BELOW CLEANUP MESSAGES
===================================================================================

And in the slurm file, something like
[mpiexec@aci-117.chtc.wisc.edu] HYDT_bscu_wait_for_completion (./tools/bootstrap/utils/bscu_wait.c:76): one of the processes terminated badly; aborting

**Comment by henry-wu** [comment]
Yes, this error is probably due to ACI.  But MAST should still catch it and just resubmit the job.
